packages/aragon-cli/src/commands/apm_cmds/util/generate-artifact.js

```diff
@@ -47,18 +58,16 @@ async function deprecatedFunctions(apm, artifact, web3, reporter) {
             deprecatedOnVersion = []
           }
         } else {
-          // TODO: (Gabi) Handle warning messages
-          // reporter.warning(
-          //   `Cannot find artifacts for version ${version.version} in aragonPM repo. Please make sure the package was published and your IPFS or  HTTP server are running.`
-          // )
-          return deprecated
+          reporter.warning(
+            `Cannot find artifacts for version ${version.version} in aragonPM repo. Please make sure the package was published and your IPFS or HTTP server are running.`
+          )
```

Comment:
We may want to also ask the user if they'd like to abort here... if I wanted deprecated methods but the old CID wasn't fetchable I'd want to stop the publishing process.